notebooks/writing_a_trace_transform_cpu_offloading.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        variable_to_symbol (dict): A dictionary mapping each variable to the  symbol where it is first/last used based on <code>first_used</code> argument.
</pre><p>ditto (indent)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>